opentreemap/treemap/tests/test_udfs.py

```diff
@@ -1306,6 +1306,61 @@ def test_delete_cudf_deletes_mobile_api_field_group(self):
         self.assertEquals(1, len(updated_instance.mobile_api_fields))
 
 
+class ProtectedUDFTestCase(OTMTestCase):
+    def setUp(self):
+        User._system_user.save_base()
+
+        self.instance = make_instance()
+        create_stewardship_udfs(self.instance)
```

Comment:
The `udf_create` helper raises a `ValidationException` (something to do with mobile API fields) if the instance doesn't have stewardship UDFs. Even though we don't need stewardship UDFs for this test case, adding them resolved this error. Not sure if this is by design or not.